tests/test_termui.py

```diff
@@ -44,6 +44,16 @@ def cli():
     assert result.exception is None
 
 
+def test_choices_list_in_prompt(runner, monkeypatch):
+    @click.option('-g', type=click.Choice(['none', 'day', 'week', 'month']), prompt=True)
+    @click.command()
+    def cli():
+        pass
+
+    monkeypatch.setattr(click._termui_impl, 'isatty', lambda _: True)
+    assert '(none, day, week, month)' in runner.invoke(cli, [], input='none').output
```

Comment:
Why would that be clearer? The entire output is 'G (none, day, week, month): none\n' where the only part of this feature present in the output is '(none, day, week, month)'.

On the other hand, I only repeated the style I found in the progress bar test.

```python
    assert label in runner.invoke(cli, []).output
```